presto-geospatial/src/main/java/com/facebook/presto/plugin/geospatial/GeometryType.java

```diff
@@ -85,4 +86,12 @@ public Object getObjectValue(ConnectorSession session, Block block, int position
         Slice slice = block.getSlice(position, 0, block.getSliceLength(position));
         return deserialize(slice).asText();
     }
+
+    @Override
+    public boolean equalTo(Block leftBlock, int leftPosition, Block rightBlock, int rightPosition)
```

Comment:
Do you think it makes sense to add the `EQUALS` operator (and `HASH`) as well? It is not clear what is used where, and it can be subject to be changed.

Reply:
@arhimondr Yes, I think so. I planned to do this next (to enable GROUP BY geometry)